parts/2-ngi_pipeline/slides.md

```diff
@@ -1,10 +1,24 @@
-# NGI pipeline
+## TACA - Preprocessing
 
-## Intro to DNA sequencing
-We sequence the source code for life; DNA.
-How do you sequence DNA? What do you do with ACTG?
-Scale of data, automation, backup and clean up, why?
+First and foremost, data needs to go from the sequencer to a machine were the
+preprcessing is done. This is done in local servers because of two reasons.
 
-## TACA
-TACA starts analysis => NGI pipeline
-- configures what analsysis to start
+1. The preprocessing of the data is highly I/O intensive (converting the images
```

Comment:
I guess you're not going into these details but BCL:s are not technically images, no? More like intensities is what I remembered.

If this is what you meant - please ignore this comment!